js/index.js

```diff
@@ -44,3 +44,34 @@ messageForm.addEventListener("submit", function (event) {
   newMessage.appendChild(removeButton);
   messageList.appendChild(newMessage);
 });
+// AJAX
+// New XMLHttpRequest object
+let githubRequest = new XMLHttpRequest();
+// Open a request
+githubRequest.open("GET", "https://api.github.com/users/SokolovskayaE/repos");
+// Send the request
+githubRequest.send();
+
+// Add a "load" event listener
+githubRequest.addEventListener("load", (event) => {
+  if (githubRequest.status === 200) {
+    // Successful HTTP request
+    let repositories = JSON.parse(githubRequest.response); // Parse the response and store it in a variable named repositories
+    console.log(repositories);
+
+    // Select the #projects section by id and store it in a variable named projectSection
+    let projectSection = document.getElementById("projects");
+    // Query the projectSection (instead of the entire document) to find the <ul> element and store it in a variable named projectList
+    let projectList = projectSection.querySelector("ul");
+    // For loop to iterate over your repositories Array, starting at index 0
+    for (let i = 0; i < repositories.length; i++) {
```

Comment:
Could use for (let repo of repositories) here.

Reply:
Thanks! Made the change and pushed another commit.